test_aks.py

```diff
@@ -3,18 +3,44 @@
 from pageobjects import WelcomePage
 from pageobjects import SideBar
 from pageobjects import Cluster
+from pageobjects import Variables
 import pytest
 
 
-def test_simpleFlow():
+@pytest.fixture
+def variables_values():
     config = configparser.RawConfigParser()
     config.read('test.properties')
+    keys = ["subscription_id", "resource_group", "location", "client_id", "client_secret", "tenant_id",
+            "ssh_username", "ssh_public_key", "cluster_admin_password", "uaa_admin_client_secret", "dns_zone_name", "cap_domain", "email"]
+    return dict(map(lambda x: (x, config.get('variables', x)), keys))
+
+
+@pytest.fixture
+def login():
+    config = configparser.RawConfigParser()
+    config.read('test.properties')
```

Comment:
The README mentions `.properties` but we are reading from `test.properties`

Reply:
done